File: messaging/v1/service/update-service-update/update-service-update.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

service = @client.messaging
                 .services('MG2172dd2db502e20dd981ef0d67850e1a')
                 .update(inbound_request_url: 'http://www.example.com')

puts service.friendly_name

File: messaging/v1/service/update-service-update/update-service-update.5.x.php
<?php

// Update the path below to your autoload.php,
// see https://getcomposer.org/doc/01-basic-usage.md
require_once '/path/to/vendor/autoload.php';

use Twilio\Rest\Client;

// Find your Account Sid and Auth Token at twilio.com/console
$sid    = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token  = "your_auth_token";
$twilio = new Client($sid, $token);

$service = $twilio->messaging->v1->services("MG2172dd2db502e20dd981ef0d67850e1a")
                                 ->update(array(
                                              "inboundRequestUrl" => "http://www.example.com"
                                          )
                                 );

print($service->friendlyName);